src/test/java/org/apache/sling/jcr/contentloader/it/ContentloaderTestSupport.java

```diff
@@ -196,8 +190,8 @@ protected void waitForContentLoaded() throws Exception {
      */
     protected void waitForContentLoaded(long timeoutMsec, long nextIterationDelay) throws Exception {
         Awaitility.await("waitForContentLoaded")
-            .atMost(Duration.ofMillis(timeoutMsec))
```

Comment:
If switching back to awaitility 4.x then these two changed lines should be reverted back to how it was before.